mottak/src/main/kotlin/no/nav/melosys/soknadmottak/soknad/SoknadSkjemaOversetter.kt

```diff
@@ -248,4 +248,13 @@ object SoknadSkjemaOversetter {
         return if (erRådgivningsfirmaFullmektig(innhold)) innhold.fullmakt.fullmektigVirksomhetsnavn
         else innhold.arbeidsgiver.virksomhetsnavn
     }
+
+    fun avklarKvitteringMottaker(søknad: Soknad): String {
+        val innhold = kotlinXmlMapper.readValue(søknad.innhold, MedlemskapArbeidEOSM::class.java).innhold
+        return if (arbeidstakerHarGittFullmakt(innhold)) {
+            innhold.fullmakt.fullmektigVirksomhetsnummer
+        } else {
+            innhold.arbeidstaker.foedselsnummer
+        }
```

Comment:
Ser du har spurt om dette på Slack. Venter du på en avklaring?

Reply:
For meg var dette en opplagt måte å løse behovet. Jeg lurte på om det fantes en annen måte gjennom Altinn sitt API. Det finnes ikke.